apps/src/templates/rubrics/rubrics.module.scss

```diff
@@ -332,25 +329,19 @@ button.rubricHeaderTab {
 .aiToken {
   display: flex;
   height: 20px;
-  padding: 4px 8px;
+  padding: 3px 8px 0;
   justify-content: center;
   align-items: center;
   gap: 4px;
-
   border-radius: 30px;
-  border: 1px solid $product_information_default;
-  background: $lightest_cyan;
+  background: #3cfff8;
```

Comment:
Just checking... should this be a color from our `color` file?

Reply:
That color isn't in the file yet. I suppose I could add it? Wasn't sure who managed that.